xsd/netex_part_5/part5_fm/netex_nm_individualTraveller_version.xsd

```diff
@@ -368,6 +373,52 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether  travelling with Pet.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="reviews" type="reviews_RelStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Reviews associated to the passenger</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
 		</xsd:sequence>
 	</xsd:group>
+	<!-- ====  REVIEWS.  ============================================ -->
+	<xsd:element name="Review" type="ReviewStructure">
+		<xsd:annotation>
+			<xsd:documentation>A review of a diver or of a traveler.</xsd:documentation>
+		</xsd:annotation>
+	</xsd:element>
+	<xsd:complexType name="reviews_RelStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for collection of reviews.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element ref="Review" maxOccurs="unbounded"/>
+		</xsd:sequence>
+	</xsd:complexType>
+	<xsd:complexType name="ReviewStructure">
+		<xsd:annotation>
+			<xsd:documentation>Type for Review.</xsd:documentation>
+		</xsd:annotation>
+		<xsd:sequence>
+			<xsd:element name="Date" type="xsd:dateTime" minOccurs="0">
```

Comment:
If it is named a Date why do you use _xsd:dateTime_?

Reply:
right... I'll change it